packages/features/booking-audit/lib/actions/AttendeeRemovedAuditActionService.ts

```diff
@@ -0,0 +1,82 @@
+import { z } from "zod";
+
+import { StringArrayChangeSchema } from "../common/changeSchemas";
+import { AuditActionServiceHelper } from "./AuditActionServiceHelper";
+import type { IAuditActionService, TranslationWithParams } from "./IAuditActionService";
+
+/**
+ * Attendee Removed Audit Action Service
+ * Handles ATTENDEE_REMOVED action with per-action versioning
+ */
+
+// Module-level because it is passed to IAuditActionService type outside the class scope
+const fieldsSchemaV1 = z.object({
+    removedAttendees: StringArrayChangeSchema,
+});
```

Comment:
what does `removedAttendees: StringArrayChangeSchema` in  AttendeeRemovedAuditActionService do? The schema here just tracks new attendee list.